sources/packages/api/src/types/config.ts

```diff
@@ -87,3 +88,12 @@ export interface CRAIntegrationConfig {
   programAreaCode: string;
   environmentCode: string;
 }
+
+export interface ATBCIntegrationConfig {
+  ATBCClientCreateEndpoint: string;
```

Comment:
Usually I believe that we are creating the env variables for the root URL and defining the endpoints based on the root URL. Is there a need to have the endpoints also as env variables?

Reply:
since it's an external API endpoint, there may be a chance of change in endpoint right, and if there is any scenario, without any code changes, we can update the endpoint. if that doesn't make sense, then I can update the code

Reply:
I got the point but, if they change the endpoints, we will be changing at least our CICD to accommodate the new endpoint names. When we go for production we should not expect such changes to happen, otherwise, it would be a major backward compatibility issue. I would recommend keeping just the root URL as a parameter that is what is going to be different per environment. Always open to chat about it if you prefer :smile: